robottelo/hosts.py

```diff
@@ -257,6 +257,24 @@ def subscription_manager_status(self):
     def subscription_manager_list(self):
         return self.execute('subscription-manager list')
 
+    def subscription_manager_get_pool(self, sub_list=[]):
+        pool_ids = []
+        for sub in sub_list:
+            result = self.execute(
+                f'subscription-manager list --available --pool-only --matches="{sub}"'
+            )
+            result = result.stdout
+            result = result.split('\n')
+            result = ' '.join(result).split()
+            pool_ids.append(result)
+        return pool_ids
+
+    def subscription_manager_attach_pool(self, pool_list=[]):
```

Comment:
Add a one docstring, it could be small as well.

Reply:
Done